StripePaymentSheet/StripePaymentSheetTests/PaymentSheet/UpdatePaymentMethodViewControllerSnapshotTests.swift

```diff
@@ -8,9 +8,18 @@
 import StripeCoreTestUtils
 @_spi(STP) @testable import StripePaymentSheet
 @testable import StripePaymentsTestUtils
+@testable@_spi(STP) import StripeUICore
 import XCTest
 
 final class UpdatePaymentMethodViewControllerSnapshotTests: STPSnapshotTestCase {
+    override func setUp() {
+        super.setUp()
+        let expectation = expectation(description: "Load specs")
+        AddressSpecProvider.shared.loadAddressSpecs {
+            expectation.fulfill()
+        }
+        waitForExpectations(timeout: 1)
```

Comment:
Nit (maybe this wont' work since this function isn't async) `await AddressSpecProvider.shared.loadAddressSpecs()`

Reply:
unfortunately this does not seem to work :(